exe/build

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+set -e
+JEKYLL_ENV=production bundle exec jekyll build --config jekyll.yml --profile
```

Comment:
This will run in production mode locally (and load analytics etc). Should we inherit `JEKYLL_ENV` from the environment?

Reply:
It would also result in preview apps being indexed unexpectedly

Reply:
🤦 good catch! yeah, it should inherit it from the environment.